rpm/SPECS/docker-compose-plugin.spec

```diff
@@ -35,14 +35,7 @@ Docker Compose V1 ('docker-compose').
 
 %build
 pushd ${RPM_BUILD_DIR}/src/compose
-    GO111MODULE=on go mod download
-    GO111MODULE=on \
-    CGO_ENABLED=0 \
-        go build \
-            -trimpath \
-            -ldflags="-s -w -X github.com/docker/compose/v2/internal.Version=%{_compose_version}" \
-            -o "bin/docker-compose" \
-            ./cmd
+    make VERSION=%{_compose_version} build
```

Comment:
Need to check if we can immediately install it in the right location, instead of doing that in `%install` (I seem to recall the RPM build expects it to be done there, but maybe it's not needed.

Reply:
The only thing I was wondering while making this PR was; should the makefile have a `make install`? That would be a closer match to the recommendation for using `DESTDIR`, but perhaps that also complicate things if we need to call a different "build" target (cross vs build?)

Reply:
Oh, my bad; forgot  to update the `install` below to take the new build location into account;

```
+ install -D -p -m 0755 bin/docker-compose /root/rpmbuild/BUILDROOT/docker-compose-plugin-9c23e015228d68bf1038343781d8a84283a9a545-0.el7.aarch64/usr/libexec/docker/cli-plugins/docker-compose
install: cannot stat 'bin/docker-compose': No such file or directory
error: Bad exit status from /var/tmp/rpm-tmp.WBAF2z (%install)
    Bad exit status from /var/tmp/rpm-tmp.WBAF2z (%install)
```